templates/include/common/event.1.parquetmr.txt

```diff
@@ -0,0 +1,13 @@
+(TUPLE) {
```

Comment:
Not liking the partial object template unfortunately the other options aren't much better with this simple templating.

I would get rid of the extra indent spacing (just go with the standard 2) as it adds little value and won't align properly in the two different contexts where the template is used.  Added #105 to address formatting in general.